addons/im_livechat/static/tests/tours/im_livechat_session_history_open.js

```diff
@@ -0,0 +1,44 @@
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add("im_livechat_session_history_open", {
+    steps: () => [
+        {
+            trigger: "body",
+            run: "press ctrl+k",
+        },
+        {
+            trigger: ".o_command_palette_search input",
+            run: "fill /",
+        },
+        {
+            trigger: ".o_command_palette_search input",
+            run: "fill Live Chat",
+        },
+        {
+            trigger: ".o_command:contains(Sessions History)",
+            run: "click",
+        },
+        {
+            trigger: ".o_switch_view[data-tooltip='List']",
+            run: "click",
+        },
+        {
+            trigger: ".d-block:contains('Participants')",
+            run: "click",
+        },
```

Comment:
Is there a particular reason to switch to list view rather than clicking on a session from the kanban page directly?